airflow/contrib/operators/bigquery_operator.py

```diff
@@ -118,7 +118,8 @@ def __init__(self,
                  query_params=None,
                  labels=None,
                  priority='INTERACTIVE',
-                 time_partitioning={},
+                 time_partitioning=None,
+                 api_resource_configs=None,
```

Comment:
Add this parameter in docstring with example.

Reply:
@kaxil done